src/Frontend-GNOME/Views/MessageTextView.cs

```diff
@@ -203,6 +205,41 @@ public void ApplyConfig(UserConfig config)
             }
 
             _BufferLines = (int) config["Interface/Notebook/BufferLines"];
+            ResizeEmoji();
+        }
+
+        void ResizeEmoji()
+        {
+            int width, height;
+            using (var layout = CreatePangoLayout(null)) {
+                layout.GetPixelSize(out width, out height);
+            }
+
+            var iter = Buffer.StartIter;
+            while (!iter.IsEnd) {
+                var mark = Buffer.CreateMark(null, iter, true);
```

Comment:
This is expensive, store Buffer in a local variable (else it makes a GTK+ call to retrieve the buffer)